SignalServiceKit/src/Messages/Attachments/TSAttachmentStream.m

```diff
@@ -749,9 +749,9 @@ - (nullable OWSLoadedThumbnail *)loadedThumbnailSmallSync
     __block OWSLoadedThumbnail *_Nullable asyncLoadedThumbnail = nil;
     OWSLoadedThumbnail *_Nullable syncLoadedThumbnail = nil;
     syncLoadedThumbnail = [self loadedThumbnailWithThumbnailDimensionPoints:kThumbnailDimensionPointsSmall
-        success:^(OWSLoadedThumbnail *asyncLoadedThumbnail) {
```

Comment:
This is the fix - we were assigning to the local param, which shadowed the name of the outer variable.